x/vote/exported/types.go

```diff
@@ -85,8 +86,8 @@ func (m PollKey) Validate() error {
 		return fmt.Errorf("missing module")
 	}
 
-	if m.ID == "" {
-		return fmt.Errorf("missing poll ID")
+	if err := utils.ValidateString(m.ID); err != nil {
+		return sdkerrors.Wrap(err, "invalid recipient address")
```

Comment:
This is where we check `pollKey.ID`. Also, the log message should be updated.
```suggestion
		return sdkerrors.Wrap(err, "invalid poll key ID")
```